test/pylib/minio_server.py

```diff
@@ -93,7 +95,7 @@ async def mc(self, *args, ignore_failure=False, timeout=0):
             else:
                 break
 
-    def _anonymous_public_policy(self):
+    def _bucket_policy(self):
         # the default anonymous public policy does not allow us to access
```

Comment:
This comment is not relevant any more, and suggests that the code that follows it also isn't relevant any more?

Reply:
Well, it is relevant. The comment explains why we don't apply default policies and instead create our own. This PR makes this policy applied to a specific user, not to anonymous access.

Only the word "anonymous" no longer applies, probably